### PR TITLE
styled challenge cards

### PR DESCRIPTION
**Pull request by UlrikeWerner**

### CODE REVIEW

File: src/components/Challenge/Style/StyledChallengeContainer.js [hunk 1]
```diff
@@ -1,33 +1,22 @@
...
-	}
-
-	h2 {
-		font-size: 1.25rem;
-	}
-
-	p {
-		font-size: 0.9rem;
-	}
+	background: var(--dlGreyGradient);
```
Comment (p-enz) [inline src/components/Challenge/Style/StyledChallengeContainer.js:11]: Like that you use css custom props

File: src/components/Challenge/index.js [hunk 2]
```diff
@@ -87,6 +89,6 @@ export default function Challenge({
 					start
 				</Button>
 			)}
-		</ChallengeContainer>
+		</StyledChallengeContainer>
```
Comment (p-enz) [inline src/components/Challenge/index.js:92]: Like the prefix Styled !

File: src/styles.js [hunk 3]
```diff
@@ -8,13 +8,15 @@ export const GlobalStyle = createGlobalStyle`
 	}
 
 	:root{
-		--bgColor: #F6F6F6;
+		--offWhite: #F6F6F6;
+		--bgColor: var(--offWhite);
```
Comment (p-enz) [inline src/styles.js:12]: do you need both 
`--offwhite and --bgColor`? because they have the same value :)

**Review by p-enz — APPROVED** [review]
Like your styling so far. I can imagine that it will be a awesome feeling to complete the first challenge !! Good work

File: src/styles.js (hunk 3 above)
Comment (UlrikeWerner) [reply src/styles.js:12]: later

**Pull request merged**